.github/workflows/push-to-testnet-draft.yml

```diff
@@ -8,8 +8,6 @@ jobs:
     runs-on: ubuntu-latest
```

Comment:
i think repository_dispatch or workflow_dispatch actions have to be in the master branch to work

Reply:
See my comment above.